subprojects/core/src/main/resources/org/gradle/reporting/report.js

```diff
@@ -65,6 +65,29 @@
         }
     }
 
+    function initClipboardCopyButton() {
+        document.querySelectorAll(".copy-btn").forEach((button) => {
+            const container = button.parentElement;
+            const pre = container.querySelector("pre");
+
+            button.addEventListener("click", () => {
+                const text = pre.innerText.trim();
```

Comment:
🤔 Is this the best way to select this element?  Maybe it should be given an `id` so that we can select it directly making selection more efficient, the code easier to understand, and the process harder to accidentally break with changes to the report.